### DISCUSSION TITLE
LGGT

### DISCUSSION THREAD

**Discussion by igghcz** [berarma/oversteer#162] [General]
Hello.
I tried Logitech GT in TD Ferrari Racing Legends and in Win the brake is mapped to the -RZ axis, but in Linux it is mapped to the -Z axis. The GT game profile works in Win but not in Lin. But it can be redefined.

On Win I use (via Win Control Panel) fixed parameters 366 degrees, FFB on, Overall 12+%, Spring 27%, Damper 27%, Enable Centering 25%, Allow Game To Adjust Settings NO. 

On Lin and use new-lg4ff and set this Win parametesrs via Oversteer and then use read only attributes /sys/bus/hid/drivers/logitech/XXXX:XXXX:XXXX.XXXX/ files. (Setting Allow Game To Adjust Settings: NO via Oversteer would be fine.)

But the effects in game are completely different. And setting values for autocenter, damper_level, friction_level maybe does nothing. The gain setting works as expected.

**Comment by berarma** [comment]
For a fair comparison you should disable any settings, or set them to their default values, in the Win Control Panel. The "Allow game to Adjust settings" isn't available on Linux, so you should allow the game to change settings to compare equally. This feature isn't planned and it wouldn't probably be very useful since games can only change rotation range and the FFB gain anyway. If changing values for spring, damper and friction doesn't change anything it's probably because the game doesn't use those effects.
